server/routes/userRoutes.js

```diff
@@ -6,4 +6,6 @@ const router = express.Router();
 
 router.post('/register', userController.register);
 
+router.post('/login', userController.login);
```

Comment:
I know this isn't convention, nor is it what we're doing on this project...but food for thought. Handling returns to client here put all of them in once place, so they're easily reviewable when checking problems between the front and back end. You can see exactly what's going to the client and how it's getting there without digging through all of your controller logic.